vsutil.py

```diff
@@ -166,3 +166,12 @@ def is_image(filename: str) -> bool:
     Returns true if a filename refers to an image.
     """
     return mimetypes.types_map.get(os.path.splitext(filename)[-1], "").startswith("image/")
+
+
+def clip_sum(*clips: List[vs.VideoNode]) -> vs.VideoNode:
```

Comment:
Why not use an iterable like `sum` itself? That makes their signatures almost equivalent (except for `start`, which is the sole reason you added this anyway). Code will get a bit more complex as a result, however.

Reply:
Isn’t this more user-friendly? I like the varargs here because it allows you to write `clip_sum(clipa, clipb, clipc)` rather than `clip_sum([clipa, clipb, clipc])`.

Reply:
Depends on what you are going for. I'd prefer signature equivalence with the `sum` function it is based on over saving two characters. (And it's actually more powerful, albeit seldom relevant.)